### PR TITLE
[Do not merge] [WIP] [Spike] Refactor dashboard graphs

### PR DESCRIPTION
**Pull request by govindkrjoshi**
**Story card:** [sc-XXXX](URL)

## Because

Enter the reason for raising this PR...

## This addresses

Enter the details of what all this covers...

## Test instructions

Enter detailed instructions for how to test this PR...

### CODE REVIEW

File: app/views/reports/regions/diabetes.html.erb [hunk 1]
```diff
@@ -8,66 +8,9 @@
...
-            <span data-fasting=""></span> Fasting
-            <span data-hba1c=""></span> HbA1c
-          </p>
-        </div>
-      </div>
-    </div>
-  </div>
+  <%= render 'reports/regions/diabetes/bs_below_200_card',
+             with_ltfu: @with_ltfu,
+             period: @period %>
```
Comment (prabhanshuguptagit) [inline app/views/reports/regions/diabetes.html.erb:13]: Should pass in `@data` and `@region` explicitly as well.

File: app/views/reports/regions/diabetes/_bs_below_200_card.html.erb [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<% denominator_copy = with_ltfu ? "diabetes_denominator_with_ltfu_copy" : "diabetes_denominator_copy" %>
+<% data = pivot_period_data(@data.slice(:bs_below_200_rates, :bs_below_200_patients, :adjusted_diabetes_patient_counts)) %>
```
Comment (prabhanshuguptagit) [inline app/views/reports/regions/diabetes/_bs_below_200_card.html.erb:2]: Is this where we would [support the LTFU toggle](https://github.com/simpledotorg/simple-server/pull/3694/files#diff-1920189d420fe89ceec67f02ee365f2586073d12a5716d252b450820a8555712R766-R773)when we get to it?

File: app/views/reports/regions/diabetes/_bs_below_200_card.html.erb [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        </div>
+      </div>
+    </div>
+
+    <div>
+      <div class="p-relative px-20px mb-12px">
+        <p class="m-0px c-grey-dark c-print-black">
+          <span data-rbs-ppbs=""></span> RBS/PPBS
+          <span data-fasting=""></span> Fasting
+          <span data-hba1c=""></span> HbA1c
```
Comment (prabhanshuguptagit) [inline app/views/reports/regions/diabetes/_bs_below_200_card.html.erb:75]: Oof these need to lookup inside `breakdown`, hrm.

**Comment by govindkrjoshi** [comment]
This branch has diverged by a lot. We'll open a new PR.

**Pull request closed without merging**